### PR TITLE
[DNM] test EE tests

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
zuul.yaml

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1062
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1046

### CODE REVIEW

**Comment by Akasurde** [comment]
recheck

**Comment by Akasurde** [comment]
recheck

**Comment by tremble** [comment]
@Akasurde Please can we rename these jobs "network-ee" is misleading.  Something like "ansible-ee-" would be better.

**Comment by Akasurde** [comment]
@tremble This is an already written job and shared by multiple teams. So I think @pabelanger is the right person to answer this.

**Comment by pabelanger** [comment]
@Akasurde @tremble We can rename them, but we need to bikeshed on the format.  This is a test EE, so we likly should rename to test-ee however, it will be work to do so.

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
 botocore>=1.16.0
 boto3>=1.13.0
+coverage==4.5.4
+placebo
+mock
+pytest-xdist
+# We should avoid these two modules with py3
+pytest-mock
+# Needed for ansible.netcommon.ipaddr in tests
+netaddr
```
Comment (pabelanger) [inline requirements.txt:11]: These requirements should be removed now, and retested.

**Review by pabelanger — CHANGES_REQUESTED** [review]

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
Sorry, I caused a bug in the docker jobs over the weekend this is now fixed by: https://github.com/ansible/ansible-zuul-jobs/pull/1062

**Comment by pabelanger** [comment]
Added https://github.com/ansible/ansible-zuul-jobs/pull/1046 too

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
failure looks to be network related

**Comment by pabelanger** [comment]
recheck

**Pull request closed without merging**